arctic/_config.py

```diff
@@ -92,3 +92,9 @@ class FwPointersCfg(Enum):
 # ---------------------------
 # Configures the size of the workers pools used for async arctic requests
 ARCTIC_ASYNC_NWORKERS = os.environ.get('ARCTIC_ASYNC_NWORKERS', 4)
+
+
+# -------------------------------
+# Flag used for indicating caching levels. For now just for list_libraries.
+# -------------------------------
+ENABLE_CACHE = bool(os.environ.get('ENABLE_CACHE'))
```

Comment:
Should we change the default for this to on?

Reply:
Yeah, it's not a risky thing to turn on by default. If we are doing that then maybe we should have cache invalidation when a new library is initialized / renamed given a common flow for people might be to list_libraries on creating a new one, and it might be confusing to not see their newly created libs.

Reply:
@shashank88 Good idea.  nit: instead of basic cache invalidation, write the updated results.

Reply:
My thinking is, that we can probably have a reasonably high TTL on the cache if most mutations eventually update as well.